README.rst

```diff
@@ -185,9 +185,9 @@ To delete an Archive, you can call the ``opentok.delete_archive(archive_id)`` me
   archive.delete()
 
 You can also get a list of all the Archives you've created (up to 1000) with your API Key. This is
-done using the ``opentok.list_archives()`` method. There are two optional keyword parameters:
-``count`` and ``offset``; they can help you paginate through the results. This method returns an
-instance of the ``ArchiveList`` class.
+done using the ``opentok.list_archives()`` method. There are three optional keyword parameters:
```

Comment:
@normanargueta In the README we mention that it's `list_archives`, but the method is actually `get_archives`. Since the `list_archives` functionality does not exist, but `get_archives` does. Do you think we can create a backwards compatible method where `list_archives` would just call `get_archives` internally?

I know the API does not exist for `list_archives`, but the issue is that it was documented and might be something that developers think exist but doesn't hence why I'm calling it backwards compatible.